hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/commands/CreateTopicRepositoryCommand.java

```diff
@@ -22,7 +22,7 @@ public void execute(DatacenterBoundRepositoryHolder<TopicRepository> holder) {
     }
 
     @Override
-    public void rollback(DatacenterBoundRepositoryHolder<TopicRepository> holder) {
```

Comment:
**question:** Can we add similar logic for `TopicAlreadyExistsException`?

Reply:
rollbacks for `CreateTopicRepositoryCommand` are already disabled:
https://github.com/allegro/hermes/blob/a3cf6fd7ed7750815a260a69fd37a0193375511d/hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/commands/CreateTopicRepositoryCommand.java#L25

I believe we can revisit topic rollbacks in the future with the current knolwedge